datadog/resource_datadog_monitor.go

```diff
@@ -84,7 +84,7 @@ func resourceDatadogMonitor() *schema.Resource {
 				},
 				"priority": {
 					Description: "Integer from 1 (high) to 5 (low) indicating alert severity.",
-					Type:        schema.TypeInt,
+					Type:        schema.TypeString,
 					Optional:    true,
```

Comment:
```suggestion
					Optional:    true,
					ValidateFunc: validation.StringInSlice([]string{"", "0", "1", "2", "3", "4", "5"}, true),

```
sorry for the back and forth, we should add some validation on the string so that customers can't put in invalid values. apart from this I think we're good to go

Reply:
The issue with that is that we may allow arbitrary ranges later that orgs can configure. The API will 4XX priorities out of range based on the orgs custom range.